docs/usage.md

```diff
@@ -169,15 +169,22 @@ that can be used to access an existing cluster.
 #### Deployment of cluster
 
 If you would like to deploy new cluster you can run following command:
+
 ```bash
-run-ci -m deployment --ocsci-conf conf/path_to_config_to_be_used.yaml \
+run-ci -m deployment \
```

Comment:
I think this change is done by Martin.

Probably the rebase you did was done somehow wrong?

Reply:
@petr-balogh @mbukatov @PrasadDesala 
yes:( I will open new pr.. I will fix you comments